src/Shared/HttpSemanticConventionHelper.cs

```diff
@@ -77,15 +77,15 @@ private static bool TryGetConfiguredValues(IConfiguration configuration, [NotNul
     {
         try
         {
-            var stringValue = configuration![SemanticConventionOptInKeyName];
+            var stringValue = configuration[SemanticConventionOptInKeyName];
```

Comment:
Why remove the `!` here?

Reply:
Just didn't seem to be needed. Doesn't warn without it so not sure why it was there in the first place 🤷